pkg/controllermanager/controller/seed/dns/add.go

```diff
@@ -0,0 +1,50 @@
+// SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package dns
+
+import (
+	"cmp"
+
+	"sigs.k8s.io/controller-runtime/pkg/builder"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+
+	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+	"github.com/gardener/gardener/third_party/mock/controller-runtime/predicate"
+)
+
+// ControllerName is the name of this controller.
+const ControllerName = "seed-dns"
+
+// AddToManager adds Reconciler to the given manager.
+func (r *Reconciler) AddToManager(mgr manager.Manager) error {
+	if r.Client == nil {
+		r.Client = mgr.GetClient()
+	}
+	r.GardenNamespace = cmp.Or(r.GardenNamespace, v1beta1constants.GardenNamespace)
+
+	return builder.
+		ControllerManagedBy(mgr).
+		Named(ControllerName).
+		For(&gardencorev1beta1.Seed{}, builder.WithPredicates(r.SeedPredicate())).
+		WithOptions(controller.Options{
+			MaxConcurrentReconciles: 0,
```

Comment:
For other controllers, we never hardcode it to `0` but either to some slightly larger value or take it from a config and default to `0`. Do we want to leave it as-is, change the value or introduce a config for this Reconciler and default the value?

Reply:
I did not introduce a configuration on purpose as this is a temporary controller. I agree that a value other than `0` would be more appropriate. Does a `5` makes here?